ataka/player-cli/player_cli/ctfconfig.py

```diff
@@ -150,3 +148,7 @@ def submit_flags(flags):
         logger.error("Exception during flag submission: {} -> {}".format(str(r.status_code), str(r.text)))
         return FlagStatus.ERROR
 """
+
+
+def scrape_scoreboard():
+    return []
```

Comment:
why?


generally: shouldn't we delete this file since it's _always_ filled in by the webserver?